test_canonicaljson.py

```diff
@@ -109,13 +115,14 @@ def test_encode_pretty_printed(self):
         )
 
     def test_frozen_dict(self):
-        self.assertEqual(
-            encode_canonical_json(frozendict({"a": 1})),
-            b'{"a":1}',
-        )
-        self.assertEqual(
-            encode_pretty_printed_json(frozendict({"a": 1})), b'{\n    "a": 1\n}'
-        )
+        if frozendict_type is not None:
+            self.assertEqual(
+                encode_canonical_json(frozendict_type({"a": 1})),
+                b'{"a":1}',
+            )
+            self.assertEqual(
+                encode_pretty_printed_json(frozendict_type({"a": 1})), b'{\n    "a": 1\n}'
+            )
```

Comment:
I think it'd be slightly clearer to conditionally skip this test using https://docs.python.org/3/library/unittest.html#unittest.skipIf or similar.

Reply:
Fixed in 7ddb3b8.